fastlane/Fastfile

```
@@ -0,0 +1,84 @@
+  desc "Build Android Release AAB"
+  lane :build_release do
+    gradle(
+      project_dir: ".",
+      task: "bundleRelease",
+      properties: { "katana.flavor" => "release" },
+    )
+  end
+end
+
+platform :ios do
+  IOS_PROJECT = "app-ios/Katana.xcodeproj"
```

RuboCop (1.85.1)

[warning] Do not define constants this way within a block.

(Lint/ConstantDefinitionInBlock)